keanu-project/src/main/java/io/improbable/keanu/vertices/Vertex.java

```diff
@@ -34,6 +34,11 @@ public void setLabel(VertexLabel label) {
         this.label = label;
     }
 
+    public <V extends Vertex<T>> V labelledAs(VertexLabel vertexName) {
```

Comment:
Nit: `labelled` is UK spelling. We *usually* go with US spelling (e.g. `labeled`). I know I know it's silly and it's not just my American bias here. To be consistent we have to chose.

Reply:
done